docs/testing/unit-tests.md

```diff
@@ -0,0 +1,3 @@
+# Unit Tests
+
+TBD
```

Comment:
> Are you wanting a blurb about what and why we unit test?

It would be great for start. We can team up when I'm back from AFK and come up with comprehensive document based on the slides you shared in the link (BTW, they are great and in my opinion they should be shared with the public, too). I found recently an excellent guide which we can use as a reference: http://acco.io/a-practical-guide-to-testing-react-apps/.